### ISSUE TITLE
Trouble fitting labels inside the picture

### ISSUE THREAD

**Issue by PeterNorlindh** [JuliaGraphs/GraphPlot.jl#161]
I'm using draw(PNG("myGraph.png", 200, 200), gplot(g)) and have trouble fitting the nodelabels entirely inside the frame of the picture, see below. Is there a way to fix this?

![Graph - Label does not fit](https://user-images.githubusercontent.com/45670604/142662601-2a3792a3-722e-4133-be39-240dd778a1e7.png)

Thanks, this package is awesome!

**Comment by hdavid16** [comment]
@PeterNorlindh a quick workaround would be to use `join.(split.(labels,"_"),"\n")`. This would split your labels by underscore and then join each segment on a newline.

**Comment by hdavid16** [comment]
Another option is to use `nodelabeldist` along with `nodelabelangleoffset` to offset the labels:

As an example:
```julia
gplot(
    path_digraph(3), 
    [1,2,3], #x coord
    [1,2,3], #y coord
    nodelabel=["ABCDEFGHI","JKLMNOP","QRSTUVWXYZ"],
    nodelabeldist=[2.5,0,-2.7], #shift the first node 2.5 to the right, middle node 0 units, right node 2.7 to left
    nodelabelangleoffset=0 #label along the horizontal center line of the nodes
)
```

Will produce the plot:
<img width="344" alt="image" src="https://user-images.githubusercontent.com/9156129/181163005-e0f4b52a-4527-455f-8bfa-f575547f2376.png">

**Comment by etiennedeg** [comment]
Closed by #186
